lib/Platform/Intl/java/com/facebook/hermes/intl/DateTimeFormat.java

```diff
@@ -244,9 +245,13 @@ private void initializeDateTimeFormat(List<String> locales, Map<String, Object>
     if (JSObjects.isUndefined(timeZone)) {
       timeZone = DefaultTimeZone();
     } else {
-      String normalizedTimeZone = normalizeTimeZoneName(JSObjects.getJavaString(timeZone));
-      if (!isValidTimeZoneName(normalizedTimeZone)) {
-        throw new JSRangeErrorException("Invalid timezone name!");
+      try {
```

Comment:
This try-catch block seems unnecessary, since it just rethrows any exception thrown inside it.